TweetBook/Services/IPostService.cs

```diff
@@ -4,7 +4,7 @@ namespace TweetBook.Services
 {
     public interface IPostService
     { 
-       public Task<List<Post>> GetAllAsync();
+       public Task<IEnumerable<Post>> GetAllAsync();
```

Comment:
I only mentioned IEnumerable because it is used on projects and for your knowledge. 
You use IEnumerable when you want to loop through the items in a collection. IList is when you want to add, remove, and access the list contents out of order.
So please change to IList